Automattic/wp-calypso
# Editing Toolkit CI: Pin node version in PHPUnit job to fix failing wp-env

This is a fix for #47255 to get the PHPUnit CI job for the Editing Toolkit plugin running again.

Wp-env doesn't currently work with Node v14.15.0 due to an issue with `nodegit` not including pre-built binaries for that version (addressed in https://github.com/WordPress/gutenberg/pull/26712), so this PR pins the node version to `12.18.4` just for the phpunit job, so that wp-env will continue to work. Once the updated `wp-env` with nodegit 0.27.0 is available in npm, we should be able to remove this pinned version from the phpunit CI steps.

#### Changes proposed in this Pull Request

* Pin node version in the `phpunit` job so that `wp-env` will continue to work

***Note that this PR currently includes a change to one of the editing toolkit files — this is just so that the PR triggers a build of the plugin. I will remove that line before merging the PR***

#### Testing instructions



* Check show all checks at the bottom of this PR and take a look at the `Editing Toolkit Plugin / Run phpunit tests` step — it should have passed!

Fixes #47255
